rules/CodeQuality/Rector/Class_/AddCoversClassAttributeRector.php

```diff
@@ -148,6 +148,15 @@ private function matchExistingClassName(array $classNames): ?string
                 continue;
             }
 
+            $classReflection = $this->reflectionProvider->getClass($className);
+            if ($classReflection->isInterface()) {
+                continue;
+            }
+
+            if ($classReflection->isTrait()) {
```

Comment:
`! $classReflection->isClass()` should be enough

Reply:
there is also enum. I want to be explicit